helpers/admin/adminHelpers.ts

```diff
@@ -7,8 +8,9 @@ export const getPropertyArr = (
   deleteProps?: string[],
   dropdownChange?: Function
 ) => {
-  ;(deleteProps || []).forEach(prop => delete options[prop])
-  const keys = Object.keys(options)
+  const cloneOptions = _.cloneDeep(options)
```

Comment:
Trying to edit useState value directly again hence cloning to avoid type errors.